provider-ui/components/ProviderComponent.js

```diff
@@ -180,21 +175,20 @@ class ProviderComponent extends React.Component {
                   Login to Meshery by choosing from the available providers. Providers offer authentication,
                   session management and long-term persistence of user preferences, performance tests, service mesh adapter configurations and so on.
                 </p>
-                <b>Meshery Cloud</b>
-                  <ul>
-                    <li>Persistent sessions</li>
-                    <li>Save environment setup and adaper configuration</li>
-                    <li>Retrieve performance test results</li>
-                    <li>Free use</li>
-                  </ul>
-
-                <b>Local Provider</b>
-                  <ul>
-                    <li>Ephemeral sessions</li>
-                    <li>Environment setup not saved</li>
-                    <li>No performance test result history</li>
-                    <li>Free use</li>
-                  </ul>
+                <List>
+                {Object.keys(availableProviders).map((key) => {
+                  return (
+                    <React.Fragment>
+                      <ListItem
+                        key={availableProviders[key]['Description']}
```

Comment:
I think key be some unique value... Please confirm this because description of two providers might be same in future and hence can't act as unique value... 
I suggest you use provider's name (key variable) as key

> The best way to pick a key is to use a string that uniquely identifies a list item among its siblings. Most often you would use IDs from your data as keys

As pointed in https://reactjs.org/docs/lists-and-keys.html

Reply:
Correct, indeed a nice point! @nveenjain . I will update it.

Reply:
@Nikhil-Ladha is there any further changes you're looking to make before merge?

Reply:
Yeah, there is a small change which naveen mentioned so, once I do it. It's ready to merge.